core/src/test/java/io/confluent/rest/ApplicationTest.java

```diff
@@ -100,7 +96,7 @@ public void tearDown() throws Exception {
   @Test
   public void testParseListToMap() {
     assertEquals(
-        new HashMap(){
+        new HashMap<String, String>(){
```

Comment:
nit: There are a couple of other raw usages below.

Reply:
Fixed. Interesting that they didn't trigger unchecked warnings when compiling. Hmm.

Reply:
> What compiler warnings are you seeing? No compiler warnings for me:

The ones I can think of OTOH are the ones mentioned here and the one in `Application.getServer()`, although I think there were a couple of others as well.